source/production/F0.Minesweeper.Logic.Abstractions/ILocation.cs

```diff
@@ -1,5 +1,6 @@
-﻿namespace F0.Minesweeper.Logic.Abstractions
+namespace F0.Minesweeper.Logic.Abstractions
 {
+	// Discuss if this really should be an interface and not just a concrete class/record
```

Comment:
to discuss

Reply:
we dicussed:
* remove `F0.Minesweeper.Logic.Abstractions.ILocation` interface
* remove `F0.Minesweeper.Components.Location` record
* move and use `F0.Minesweeper.Logic.Location` record from `F0.Minesweeper.Logic` to `F0.Minesweeper.Abstractions` assembly

Reply:
Change is implemented!